examples/list.rs

```diff
@@ -202,80 +188,159 @@ fn run_app<B: Backend>(
 }
 
 fn ui(f: &mut Frame, app: &mut App) {
+    // Create a space for header
+    let vertical = Layout::vertical([Constraint::Length(2), Constraint::Percentage(100)]);
+    let [header_area, rest_area] = f.size().split(&vertical);
+
     // Create two chunks with equal horizontal screen space
-    let horizontal = Layout::horizontal([Constraint::Percentage(50), Constraint::Percentage(50)]);
-    let [item_list_area, event_list_area] = f.size().split(&horizontal);
+    let horizontal = Layout::horizontal([Constraint::Percentage(50), Constraint::Max(15)]);
+    let [item_list_area, event_list_area] = rest_area.split(&horizontal);
+
+    // Create two chunks with equal vertical screen space. One for the list and the other for the
+    // info block.
+    let vertical = Layout::vertical([Constraint::Percentage(50), Constraint::Percentage(50)]);
+    let [upper_item_list_area, lower_item_list_area] = item_list_area.split(&vertical);
+
+    let header = Block::default()
+        .title("Use j k or ↓ ↑ to move, h or ← to unselect, l → to change status.")
+        .title_alignment(Alignment::Center);
+    f.render_widget(header, header_area);
+
+    render_todo(f, app, &[upper_item_list_area, lower_item_list_area]);
+
+    render_events_list(f, app, event_list_area)
+}
+
+// Renders todo list and info part
```

Comment:
It might be worth splitting this to two methods. One to render the list and the other to render the todo details.